### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oed**
Update project status.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,9 @@
...
-## Project Status - `Proof of Concept`
+## Project Status - `Slip testnet`
 
-This implementation of the Ceramic protocol is only partial and is missing multiple critical components. As development moves forward this readme will be updated with the latest project status.
-
-**Missing components:**
-* Anchor records - currently anchor records are mocked
+This implementation is ready for testing, but there is still a few things left before the Clay testnet. See the [roadmap](https://github.com/ceramicnetwork/ceramic/issues/19).
 
 ## Project Structure
```
Comment (simonovic86) [inline README.md:14]: we should update project structure, as well

Comment (oed) [reply README.md:14]: yep, feel free to open a new PR. This felt more urgent :)

File: README.md [hunk 2]
```diff
@@ -7,12 +7,9 @@
...
 Monorepo containing the Typescript implementation of the Ceramic protocol.
 
-## Project Status - `Proof of Concept`
+## Project Status - `Slip testnet`
 
-This implementation of the Ceramic protocol is only partial and is missing multiple critical components. As development moves forward this readme will be updated with the latest project status.
-
-**Missing components:**
-* Anchor records - currently anchor records are mocked
+This implementation is ready for testing, but there is still a few things left before the Clay testnet. See the [roadmap](https://github.com/ceramicnetwork/ceramic/issues/19).
```
Comment (ericelliott) [inline README.md:12]: "there is still" -> "there are still"

Comment (oed) [reply README.md:12]: Thanks!

**Pull request merged**